### PR TITLE
Task-47384: Implement news editor mobile view when creating news

### PR DESCRIPTION
**Pull request by hassenelkamel-exo**
Prior to change, news editor when creating an article is not displayed correctly in mobile view. After this change, to make news editor responsive, we have removed some buttons from `ckeditor` toolbar and we have allowed `ckeditor` toolbar switching between the two views.

### CODE REVIEW

File: services/src/main/resources/locale/portlet/news/News_en.properties [hunk 1]
```diff
@@ -15,6 +15,7 @@ news.composer.btn.cancel=Cancel
 news.composer.btn.remove=Remove
 news.composer.summary=Summary
 news.composer.createNews=Write an article for {0}
+news.create.createNews=Write an article
```
Comment (azayati) [inline services/src/main/resources/locale/portlet/news/News_en.properties]:
```suggestion
news.composer.mobile.createNews=Write an article
```

File: webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue [hunk 2]
```diff
@@ -15,45 +15,55 @@
...
+            width="40"
+            min-width="40"
+            max-width="40"
+            size="40"
+            class="mx-3 my-auto spaceAvatar space-avatar-header">
+            <v-img src="/news/images/news.png" />
+          </v-avatar>
+          <div class="d-flex flex-grow-1 flex-column my-auto align-left">
+            <span class="flex-row text-truncate subtitle-1">{{ newsFormTitle }}</span>
+            <span v-if="spaceDisplayName " class="flex-row subtitle-2 text-capitalize text-truncate text-light-color spaceName">{{ spaceDisplayName }}</span>
```
Comment (azayati) [inline webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue]:
```suggestion
            <span v-if="spaceDisplayName" class="flex-row subtitle-2 text-capitalize text-truncate text-light-color spaceName">{{ spaceDisplayName }}</span>
```

File: webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue [hunk 3]
```diff
@@ -15,45 +15,55 @@
...
+      <div class="newsComposerToolbar">
+        <div class="d-flex flex-row py-2">
+          <v-avatar
+            height="40"
+            min-height="40"
+            width="40"
+            min-width="40"
+            max-width="40"
+            size="40"
+            class="mx-3 my-auto spaceAvatar space-avatar-header">
```
Comment (azayati) [inline webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue]: why using spaceAvatar space-avatar-header css classes here, can we define another css classes even if we have the same definition ?

File: webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue [hunk 4]
```diff
@@ -422,6 +437,12 @@ export default {
         },
         on: {
           instanceReady: function(evt) {
+            const topButton3 = document.getElementById('cke_14');
```
Comment (azayati) [inline webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue]: can you rename topButtons with significative names ?

File: webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue [hunk 5]
```diff
@@ -852,6 +878,30 @@ export default {
     },
     getString(body) {
       return new DOMParser().parseFromString(body, 'text/html').documentElement.textContent.replace(/&nbsp;/g, '').trim();
+    },
+    changeView() {
+      const topButton = document.getElementById('cke_7');
```
Comment (azayati) [inline webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue]: can you rename topButtons with significative names ?

File: webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue [hunk 6]
```diff
@@ -128,25 +138,6 @@
         v-model="news.attachments"
         @HideAttachmentsDrawer="onHideAttachmentsDrawer"
         @uploadingCountChanged="setUploadingCount" />
-      <!-- The following bloc is needed in order to display the pin confirmation popup -->
```
Comment (azayati) [inline webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue:131]: we need also to clean the other references to publish/unpublish except publishing drawer

File: webapp/src/main/webapp/skin/less/news.less [hunk 7]
```diff
@@ -590,118 +590,12 @@
     }
   }
 
-  .newsComposerActions {
+  .newsComposerToolbar {
```
Comment (azayati) [inline webapp/src/main/webapp/skin/less/news.less:593]: we need to verify that all references to publish/unpublish except publishing drawer are cleaned for news composer

**Review by azayati — CHANGES_REQUESTED** [review]
See review

File: webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue (hunk 3 above)
Comment (hassenelkamel-exo) [reply webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue]: i don't use this classes i will remove it.

**Review by azayati — DISMISSED** [review]
LGTM after QA validation

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**